.gitignore

```diff
@@ -6,3 +6,6 @@ build*
 *.bak
 src/mlpack/core/util/gitversion.hpp
 src/mlpack/core/util/arma_config.hpp
+
+.idea
```

Comment:
I tried coding with [Clion IDE](https://www.jetbrains.com/clion/).
Seems like Jetbrain IDEs all create .idea files. I will roll this back in the next pull request :)